config/stage.js

```diff
@@ -34,4 +34,6 @@ module.exports = {
 
   // https://sentry.prod.mozaws.net/operations/addons-frontend-amo-stage/
   publicSentryDsn: 'https://8f0a256ee2c345608510155edafb71f7@sentry.prod.mozaws.net/182',
+
+  enableFeatureUseAdzerkForSponsoredShelf: true,
```

Comment:
I am wondering why we want to only enable this feature on stage?

Reply:
Because my intent is to cherry-pick this commit, which will enable adzerk on stage immediately. As discussed in the standup today, we want to do that in order to give QA a bit more time to test it on stage. A further issue to enable it on prod (or rather everywhere) will be opened, and then the commit for that will be added to next week's tag.

Reply:
Further to the discussion on matrix, I don't think I'll be pushing this until after the push, so I guess it does make sense to just enable it everywhere. I'll update the issue and the PR.

Reply:
Yeah, we usually don't do that so I was a bit surprised. Pushing a one-off tag after tomorrow's push sounds good.